bndtools.core/src/bndtools/wizards/bndfile/NewBndFileWizardPage.java

```diff
@@ -57,8 +57,14 @@ protected boolean validatePage() {
         IResource container = ResourcesPlugin.getWorkspace().getRoot().findMember(containerPath);
         IProject project = container.getProject();
 
-        if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
-            error = Messages.NewBndFileWizardPage_errorReservedFilename;
+        try {
+            if (project.hasNature(BndtoolsConstants.NATURE_ID)) {
+                if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
+                    error = Messages.NewBndFileWizardPage_errorReservedFilename;
```

Comment:
Bah, yeah I was testing thing sin Bndtools 3.2 when I took that
screenshot.  Who is putting the "file name is reserved"?   that seems
unfriendly

On Wed, Sep 7, 2016 at 1:14 PM, Sean Bright <notifications@github.com>
wrote:

> In bndtools.core/src/bndtools/wizards/bndfile/NewBndFileWizardPage.java
> <https://github.com/bndtools/bndtools/pull/1526#discussion_r77876016>:
>
> > @@ -57,8 +57,14 @@ protected boolean validatePage() {
> >          IResource container = ResourcesPlugin.getWorkspace().getRoot().findMember(containerPath);
> >          IProject project = container.getProject();
> >
> > -        if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
> > -            error = Messages.NewBndFileWizardPage_errorReservedFilename;
> > +        try {
> > +            if (project.hasNature(BndtoolsConstants.NATURE_ID)) {
> > +                if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
> > +                    error = Messages.NewBndFileWizardPage_errorReservedFilename;
>
> @gamerson <https://github.com/gamerson>, this is what I see in latest:
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/114601/18323398/689265ba-7505-11e6-9b28-ed5f8ebddade.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/bndtools/bndtools/pull/1526/files/88248333855513e87035a943f77dfacf7ef30323#r77876016>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAkVFbEonLqBKbE8t34siZ1ZzWN7AdnPks5qnv8PgaJpZM4J3JtT>
> .
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com